commands/config_e2e_test.go

```diff
@@ -111,3 +115,60 @@ func (s *MmctlE2ETestSuite) TestConfigSetCmd() {
 		s.Require().Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestConfigEditCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForSystemAdminAndLocal("Edit a key in config", func(c client.Client) {
+		printer.Clean()
+
+		// check the value before editing
+		args := []string{"ServiceSettings.EnableSVGs"}
+		err := configGetCmdF(c, &cobra.Command{}, args)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().False(*printer.GetLines()[0].(*bool))
+		s.Require().Len(printer.GetErrorLines(), 0)
+		printer.Clean()
+
+		// create a shell script to edit config
+		content := `#! /bin/bash
+sed -i'old' 's/\"EnableSVGs\": false/\"EnableSVGs\": true/' $1
+rm $1'old'`
+
+		file, err := ioutil.TempFile(os.TempDir(), "config_edit_*.sh")
+		s.Require().Nil(err)
+		defer func() {
+			file.Close()
+			os.Remove(file.Name())
+			resetCmd := &cobra.Command{}
+			resetCmd.Flags().Bool("confirm", true, "")
+			s.Require().Nil(configResetCmdF(c, resetCmd, []string{"ServiceSettings"}))
+		}()
```

Comment:
If we use `s.th.App` utilities we may only need to close the file.
```suggestion
		defer file.Close()
```